### PR TITLE
Bump bugsnag-cocoa to v6.4.0

### PR DESCRIPTION
**Pull request by kstenerud**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Changed
 
+- (react-native): Update bugsnag-cocoa to v6.4.0
```
Comment (nickdowell) [inline CHANGELOG.md:7]: @bengourley looks like a RN release containing bugsnag-cocoa 6.3.0 never went out?!

If so, we should merge the changelog entries, and be sure to also include a summary of the changes in bugsnag-cocoa 6.4.0

**Review by nickdowell — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bengourley) [reply CHANGELOG.md:7]: Yeah we held off on that while waiting for the dust to settle on the release stage validation stuff.

Comment (bengourley) [reply CHANGELOG.md:7]: @kstenerud it would be good to merge the changelog entries please 🙏

**Review by bengourley — APPROVED** [review]
All good provided the changelog entries are merged.

File: CHANGELOG.md (hunk 1 above)
Comment (kstenerud) [reply CHANGELOG.md:7]: Added under this line

**Review by nickdowell — APPROVED** [review]

**Comment by kstenerud** [comment]
Holding off on this for now. Do not merge.

**Pull request closed without merging**